backend/pong/ws/tournament/async_db_serivice.py

```
@@ -122,23 +215,18 @@
-        if isinstance(e.detail, list):
-            return UpdateTournamentResult.error({"ValidationError": e.detail})
-        return UpdateTournamentResult.error(e.detail)
+        return UpdateTournamentResult.error(handle_validation_error(e))
 
     except tournament_models.Tournament.DoesNotExist as e:
-        logger.error(f"DoesNotExist: {e}")
         return UpdateTournamentResult.error(
-            {
-                "DoesNotExist": f"Tournament object does not exists. Details: {str(e)}."
-            }
+            handle_does_not_exist_error(
```

Ruff (0.8.2)

Undefined name `handle_does_not_exist_error`

(F821)